src/app/content/hooks/locationChange/resolveContent.spec.ts

```diff
@@ -53,11 +80,15 @@ describe('locationChange', () => {
       route: routes.content,
     };
 
-    hook = require('./resolveContent').default;
+    const resolveContent = require('./resolveContent');
+    hook = resolveContent.default;
+    resolveExternalBookReference = resolveContent.resolveExternalBookReference;
   });
 
   describe('in development', () => {
-    jest.doMock('../../../../config', () => ({...mockConfig, APP_ENV: 'development'}));
+    beforeAll(() => {
```

Comment:
Probably `beforeEach` or not needed at after all?

Reply:
not beforeEach, and also not needed, it is this way in master and works.

Reply:
It works without this in "in development", but if it is here for consistency I think it may be here

Reply:
`beforeAll` runs once before the block of tests, where `beforeEach` runs before each test. in this case `beforeAll` is fine, but it would still work with `beforeEach`